src/maheshKJun20/ValidateAtmPin.java

```diff
@@ -0,0 +1,79 @@
+package maheshKJun20;
+
+public class ValidateAtmPin {
+	
+	void validatePin(String pin) {
+		if (validateLength(pin)) {
+			if (validateDigit(pin)) {
+				if (validateNumRepitition(pin))
+					if (validateAscendingOrder(pin) || validateDescendingOrder(pin))
+						System.out.println("ATM pin " + pin + " is Valid");
+					else
+						System.out.println("Invalid pin. Order Should be Ascending or Desending");
+			}				
+		}		
+	}
+	
+	boolean validateLength(String pin) {
+		if (pin.length() == 4)
+			return true;
+		else {
+			System.out.println("Invalid pin. Length of pin: " + pin.length());
+			return false;
+		}
+	}
+	
+	boolean validateDigit(String pin) {
+		for (int index=0; index<pin.length();index++) {
+			if(!Character.isDigit(pin.charAt(index))){
+				System.out.println("Invalid pin. Invalid character in the pin '" + pin.charAt(index) + "'");
+				return false;
+			}
+		}
+		return true;
+	}
+	
+	boolean validateAscendingOrder(String pin) {
+		for (int index=0; index<pin.length();index++) {
```

Comment:
within single loop we can do it. why do we compare all numbers.
just compare first to second index, second to third and so on.. think about it .. you dont need to compare all 1 number with all previous numbers.. share your thoughts.

Reply:
@KrishnaKTechnocredits  - Yes Krishna, That would have been more appropriate solution for this. Will update the changes. 
For NumberFormatException - Realized it after submition. Will update that as well in the code.
Thanks for the Feedback.